spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -535,10 +535,8 @@ def reset_streams():
         send_notification('sct_run_batch: Run completed',
                           status_message + timing_message)
 
-    open_cmd = 'open' if sys.platform == 'darwin' else 'xdg-open'
-
-    print(f'To open the Quality Control (QC) report on a web-browser, run the following:\n'
-          f'{open_cmd} {os.path.join(path_qc, "index.html")}')
+    display_open(file=os.path.join(path_qc, "index.html"),
+                 message="To open the Quality Control (QC) report on a web-browser")
```

Comment:
Grammar nit, while we're at it.
```suggestion
                 message="To open the Quality Control (QC) report in a web-browser")
```